imagehash/tests/__init__.py

```diff
@@ -0,0 +1,53 @@
+import os
+import os.path
+import unittest
+
+from PIL import Image
+
+import imagehash
+
+
+class TestImageHash(unittest.TestCase):
```

Comment:
This is testing base class for common testing methods. Simply subclass to take advantage of this convenience.